benefits/core/templates/core/base.html

```diff
@@ -44,7 +44,7 @@
     {% endif %}
     <header role="banner" id="header">
       <a id="skip-to-content"  href="#main-content" class="d-block w-100">
-        <div class="container">{% translate "Skip to Main Content" %}</div>
+        <div class="container"><span>{% translate "Skip to main content" %}</span></div>
```

Comment:
Added this `span` here for semantic-ness.

Reply:
Can you elaborate a bit? Both `<div>` and `<span>` carry no semantic meaning on their own, the difference being denoting either block-level or inline content. Text content is allowed inside both of them... 

If there is an accessibility issue here with semantics, wouldn't a `<p>` be more appropriate?

Reply:
Semantic was the wrong word here. Though I'm not sure what the right would be. Having a border (for the focus) around a `p` or any block element (like `p` or `div`) would make it look like this, with the yellow border around the entire div: 
![image](https://github.com/cal-itp/benefits/assets/3673236/8910a27f-6d34-40f3-9239-5d500d00a595)


But a `span` is an inline-display element, so the same CSS renders the focus like this instead:
![image](https://github.com/cal-itp/benefits/assets/3673236/0a1857fa-0b62-45a9-829d-04cd9bfe82f6)

So it's not semantic-ness but more like....... the native HTML element's inherit default block-or-inline-layout-ness.